test/evergreen/compatibility_test_for_mongodb_releases.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+##############################################################################################
+# Check releases to ensure forward and backward compatibility.
+##############################################################################################
+
+###########################################################################
+# Return the most recent version of the tagged release.
+###########################################################################
+get_release()
+{ 
+	echo "$(git tag | grep "^mongodb-$1.[0-9]" | sort -n | sed -e '$p' -e d)"
+}
+
+#############################################################
+# This function will 
+#	- checkout git tree of the desired release and build it,
+#	- generate test objects.
+#
+# arg1: MongoDB tagged release number or develop branch identifier.
+#############################################################
+build_rel()
+{
+	echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+	echo "Building release: \"$1\""
+	echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+
+	git clone --quiet https://github.com/wiredtiger/wiredtiger.git "wt.$1" > /dev/null || return 1
+	cd "wt.$1" || return 1
+
+	config=""
+	config+="--enable-snappy "
+
+	case "$1" in
+	"develop")
+		branch="develop";;
+	"develop-timestamps")
```

Comment:
Would be useful to add comments here to mention develop here means the future 4.2 - the only mongodb release that supports both enabling and disabling timestamps in data format. Once 4.2 is released we'd need to update this script.